File: JAVA/Exercicio018.java
package main;

import javax.swing.JOptionPane;

public class Exercicio018 {

	public static void main(String[] args) {
		int N;
		int SOMA = 0;
		
		N = Integer.parseInt(JOptionPane.showInputDialog(""
				+ "A soma deve ir até qual numero?"));
		
		for(int ACUM = 1;ACUM <= N; ACUM++) {
			SOMA += ACUM;
		}
		JOptionPane.showMessageDialog(null, "A soma dos num. naturais" 
                + " é igual a: " + SOMA);
	}

}

File: PHP/Exercicio018.php
<!DOCTYPE html>
<html>

<body>
 <h4>A soma deve ir até qual número?</h4>
 <form action="Exercicio018.php" method="post">
    <label>Entre com um número:</label>
    <input type="text" id="N" name="N">
    <button>Calcular</button>
 </form>

<?php
  if(isset($_POST['N'])){
    $N = $_POST['N'];
    $SOMA = 0;
    for($ACUM = 1;$ACUM <= $N;$ACUM++){
        $SOMA += $ACUM;
    }
    echo "A soma dos núm. naturais é igual a: " . $SOMA;
  }
?>
</body>
</html>